Documentation/contributing/testing/e2e.rst

```diff
@@ -212,6 +217,35 @@ has been resolved):
 
 Finally, you can use the instructions from :ref:`the previous chapter<test_cilium_on_lvh>` to run and to test Cilium.
 
+Network performance test
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Cilium also provides `cilium-cli connectivity perf
+<https://github.com/cilium/cilium-cli/?tab=readme-ov-file#network-performance-test>`_
+to test the network performance of pod-to-pod communication in the same node and different nodes.
+
+To run performance test:
+
+.. code-block:: shell-session
+
+    $ cilium connectivity perf
+    ...
+    [=] Test [network-perf] [1/1]
+    ...
+
+If you want to test the network performance between specific nodes, you can label the nodes to run test:
```

Comment:
AFAIS from the source code, it picks the random nodes to deploy the test pods. it is likely that user wants to check the performance or connectivity between specific nodes, so i thought this would be useful too.

Reply:
It looks good to me; I like the fact that you show how to test on specific nodes.